addons/website_blog/controllers/main.py

```diff
@@ -1,7 +1,11 @@
 # -*- coding: utf-8 -*-
 
+from feedgen.feed import FeedGenerator
```

Comment:
@rim-odoo @antonylesuisse  werkzeug has built in helper class to generate atom feed http://werkzeug.pocoo.org/docs/0.10/contrib/atom/#werkzeug.contrib.atom.AtomFeed
but for the security reason it's not provide rss support 
http://librelist.com/browser/flask/2010/5/26/rss/#b7636992d3b5281604368ec21e81c9bf